### PR TITLE
Added Unit Tests

### PR DESCRIPTION
**Pull request by nbantar**

### CODE REVIEW

File: src/main/java/docbuddy/users/model/User.java [hunk 1]
```diff
@@ -33,4 +33,17 @@
...
     }
+
+    @Override
+    public boolean equals(Object otherUser) {
+        if (otherUser == null) return false;
+        if (!(otherUser instanceof User))return false;
+
+        User other = (User) otherUser;
+
+        if (other == this) return true;
```
Comment (nestorreloaded) [inline src/main/java/docbuddy/users/model/User.java]: I think I've learned from PMD, but should we have multiple returns on one method?

Link: http://www.javapractices.com/topic/TopicAction.do?Id=114

File: src/test/java/docbuddy/users/BaseControllerTest.java [hunk 2]
```diff
@@ -1,78 +1,31 @@
...
-        return User.builder()
-                .admin(true)
-                .doctor(true)
-                .firstName("Authenticated")
-                .lastName("User")
-                .userName("admin")
-                .password("pass")
-                .build();
-    }
+    protected final ObjectMapper mapper = new ObjectMapper();
```
Comment (nestorreloaded) [inline src/test/java/docbuddy/users/BaseControllerTest.java]: Can't use autowired annotation?

File: src/main/java/docbuddy/users/model/User.java (hunk 1 above)
Comment (nbantar) [reply src/main/java/docbuddy/users/model/User.java]: Done.

File: src/test/java/docbuddy/users/BaseControllerTest.java (hunk 2 above)
Comment (nbantar) [reply src/test/java/docbuddy/users/BaseControllerTest.java]: Done.

**Pull request merged**